hildon/maemo-gtk-compat.h

```diff
@@ -0,0 +1,66 @@
+#ifndef MAEMOGTKCOMPAT_H
```

Comment:
Please add the copyright header.
Check other headers from the source for a guide.

Reply:
fixed